### ISSUE TITLE
[OS Metrics]: Wrong query used for the number of files graph

### ISSUE THREAD

**Issue by vladzcloudius** [scylladb/scylla-monitoring#1811]
*Installation details*
Panel Name: "Number of files"
Dashboard Name: "OS Metrics"
Scylla-Monitoring Version: 3.10.1 (but most likely on any other version too)
Scylla-Version: 2021.1.x

*Description*
A query is wrong.

The used one is:
```
sum(node_filesystem_files{mountpoint="$mount_point", instance=~"$node"}) by ([[by]])
```

The one that should be used is:
```
sum(node_filesystem_files{mountpoint="$mount_point", instance=~"$node"}) by ([[by]])-sum(node_filesystem_files_free{mountpoint="$mount_point", instance=~"$node"}) by ([[by]])
```

**Comment by vladzcloudius** [comment]
@amnonh @DoronArazii FYI

**Comment by amnonh** [comment]
Duplicate of #1684 already fixed in Monitoring 4.0, please update your monitoring stack
